bin/interpreter.sh

```diff
@@ -69,7 +69,7 @@ fi
 
 . "${bin}/common.sh"
 
-ZEPPELIN_INTERPRETER_API_JAR=$(find "${ZEPPELIN_HOME}/interpreter" -name 'zeppelin-interpreter-shaded-*.jar')
+ZEPPELIN_INTERPRETER_API_JAR=$(find "${ZEPPELIN_HOME}/interpreter" -name 'zeppelin-interpreter-api-*.jar')
```

Comment:
@tokenian You reverted the previous commits. I think you may need to do rebase again.

Reply:
I use git -X ours to add my commits on top of master.Thanks a lot